include/grpcpp/ext/otel_plugin.h

```diff
@@ -170,6 +182,13 @@ class OpenTelemetryPluginBuilder {
 };
 
 namespace experimental {
+
+/// EXPERIMENTAL API
+/// TODO(): Think about whether this should under OpenTelemetryPluginBuilder
```

Comment:
I think this should definitely have "OpenTelemetry" in the name somewhere -- the current name `grpc::experiemental::MakeGrpcTraceBinTextMapPropagator()` doesn't really provide enough context to know what this is for.

One option is to make it a static method of `OpenTelemetryPluginBuilder`.  Another option is to put it in a `grpc::otel` namespace, or something similar, although that feels a little awkward, given that the OT plugin builder itself is not in such a namespace.  So the static method approach probably makes the most sense.

Reply:
Another alternative would be to keep this as a free function in the namespace grpc with the name `MakeGrpcTraceBinOpenTelemetryTextMapPropagator()`, but I think the static method in `OpenTelemetryPluginBuilder` is also fine.